<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FrmAppointment</class>
 <widget class="QWidget" name="FrmAppointment">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>155</width>
    <height>221</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Запись в другое ЛПУ</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="8" column="0">
    <widget class="QPushButton" name="btnAppoint">
     <property name="text">
      <string>Записать</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="lblReferral">
     <property name="text">
      <string>Направление:</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="lblTicket">
     <property name="text">
      <string>Дата и время приема:</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QComboBox" name="cmbTickets"/>
   </item>
   <item row="1" column="0">
    <widget class="CDbComboBox" name="cmbReferral"/>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="lblDoctor">
     <property name="text">
      <string>Врач:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QComboBox" name="cmbDoctor"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="lblSpeciality">
     <property name="text">
      <string>Специальность:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QComboBox" name="cmbSpeciality"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CDbComboBox</class>
   <extends>QComboBox</extends>
   <header>library/DbComboBox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
